3p/nameframe.max.html

```diff
@@ -5,14 +5,14 @@
   <script>
     try {
       var name_data = JSON.parse(window.name);
-      if (!name_data || !name_data['creative']) {
+      if (!name_data || !name_data.creative) {
```

Comment:
Why these changes?

Reply:
This warning isn't related to using computed property keys, it's passing something to `document.write`. Nothing we can do about that.

Reply:
There are two warnings in this file. This change addresses one of them. You're right about the other one. See [3p/nameframe.max.html](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/62a6f805a2d91db98d7bd8751b61b9171d0a8cc5/files/3p/nameframe.max.html#x8c1e62f82fa99681:1)